misc/profiles2/trekking.brf

```diff
@@ -66,7 +207,7 @@ assign probablyGood = or ispaved and isbike not isunpaved
 #
 assign turncost = if is_ldcr then 0
                   else if junction=roundabout then 0
-                  else 90
+                  else 80
```

Comment:
I slightly diminished this turncost so that cycleways around roundabout (and parallel to roads without cycleways) would not get too penalized. There is such an exemple in test suite.